app/src/main/java/com/example/cartoon/Notification.kt

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
You should remove this commented out code. from line no.1 to 73.

Reply:
done with the changes